.github/ISSUE_TEMPLATE/new-patch-release.md

```diff
@@ -18,8 +18,7 @@ Please do not remove items from the checklist
   - [ ] Write the change log into the draft release
   - [ ] Upload release artefacts generated by `prepare-release.sh` script above to the draft release
 - [ ] An OWNER runs
-     `git tag -s $VERSION`
-      and inserts the changelog into the tag description.
+     `git tag -s -m"NFD release $VERSION" $VERSION`
```

Comment:
```suggestion
     `git tag -s -m "NFD release $VERSION" $VERSION`
```

Reply:
Yeah, maybe this is cleaner. Changed as suggested